Readme.MD

```diff
@@ -139,8 +138,6 @@ Here is the list of parameters that can be environment variables or settings in
   - ```OKTA_AWS_REGION``` is the default AWS region to store with the created profile.
```

Comment:
Sounds OKTA_AWS_REGION be removed as well, or am I missing somewhere that it is used outside of the config file?

Reply:
Agreed. Done.